INPTP_AppForFixing/MainForm.cs

```diff
@@ -1,13 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Windows.Forms;
 
 namespace INPTP_AppForFixing
 {
+    public enum Action
```

Comment:
Action is name for library delegates. Use more descriptive name for this enum.

Reply:
fixed